Tools/jit/build.py

```diff
@@ -0,0 +1,695 @@
+    elif re.fullmatch(r"aarch64-.*-linux-gnu", host):
+        target = ELF("aarch64-unknown-linux-gnu", alignment=8)
+    elif re.fullmatch(r"i686-pc-windows-msvc", host):
+        target = COFF("i686-pc-windows-msvc", prefix="_")
+    elif re.fullmatch(r"x86_64-apple-darwin.*", host):
+        target = MachO("x86_64-apple-darwin", prefix="_")
+    elif re.fullmatch(r"x86_64-pc-windows-msvc", host):
+        target = COFF("x86_64-pc-windows-msvc")
+    elif re.fullmatch(r"x86_64-.*-linux-gnu", host):
+        target = ELF("x86_64-unknown-linux-gnu")
+    else:
+        raise ValueError(host)
+    return dataclasses.replace(target, debug=debug, verbose=verbose)
+
+
+def dump_header() -> typing.Generator[str, None, None]:
+    yield f"// $ {shlex.join([sys.executable, *sys.argv])}"
+    yield ""
+    yield "typedef enum {"
+    for kind in sorted(typing.get_args(schema.HoleKind)):
+        yield f"    HoleKind_{kind},"
+    yield "} HoleKind;"
+    yield ""
+    yield "typedef enum {"
+    for value in sorted(HoleValue, key=lambda value: value.name):
+        yield f"    HoleValue_{value.name},"
+    yield "} HoleValue;"
+    yield ""
+    yield "typedef struct {"
+    yield "    const uint64_t offset;"
+    yield "    const HoleKind kind;"
+    yield "    const HoleValue value;"
+    yield "    const void *symbol;"
+    yield "    const uint64_t addend;"
+    yield "} Hole;"
+    yield ""
+    yield "typedef struct {"
+    yield "    const size_t body_size;"
+    yield "    const unsigned char * const body;"
+    yield "    const size_t holes_size;"
+    yield "    const Hole * const holes;"
+    yield "} Stencil;"
+    yield ""
+    yield "typedef struct {"
+    yield "    const Stencil code;"
+    yield "    const Stencil data;"
+    yield "} StencilGroup;"
+    yield ""
+
+
+def dump_footer(opnames: list[str]) -> typing.Generator[str, None, None]:
+    yield "#define INIT_STENCIL(STENCIL) {                         \\"
+    yield "    .body_size = Py_ARRAY_LENGTH(STENCIL##_body) - 1,   \\"
+    yield "    .body = STENCIL##_body,                             \\"
+    yield "    .holes_size = Py_ARRAY_LENGTH(STENCIL##_holes) - 1, \\"
+    yield "    .holes = STENCIL##_holes,                           \\"
+    yield "}"
+    yield ""
+    yield "#define INIT_STENCIL_GROUP(OP) {     \\"
+    yield "    .code = INIT_STENCIL(OP##_code), \\"
+    yield "    .data = INIT_STENCIL(OP##_data), \\"
+    yield "}"
+    yield ""
+    yield "static const StencilGroup stencil_groups[512] = {"
+    for opname in opnames:
+        yield f"    [{opname}] = INIT_STENCIL_GROUP({opname}),"
+    yield "};"
+    yield ""
+    yield "#define GET_PATCHES() { \\"
+    for value in sorted(HoleValue, key=lambda value: value.name):
+        yield f"    [HoleValue_{value.name}] = (uint64_t)0xBADBADBADBADBADB, \\"
+    yield "}"
+
+
+def dump(stencil_groups: dict[str, StencilGroup]) -> typing.Generator[str, None, None]:
```

Comment:
```suggestion
def dump(stencil_groups: dict[str, StencilGroup]) -> typing.Generator[str]:
```

Reply:
If u wanna keep the same suggestions as before
```suggestion
def dump(stencil_groups: dict[str, StencilGroup]) -> typing.Iterable[str, None, None]:
```